### DISCUSSION TITLE
Can you trigger a command by means of a Webhook?

### DISCUSSION THREAD

**Discussion by scriptoboy** [oldschoolgg/oldschoolbot#2930] [Q&A] [answered]
Hey there!

First of all, I want to say, lovely bot! It will be a great addition to our server! So much variety!

I have a question. We are using Discord Webhooks for our server, and are trying to trigger commands using this. Unfortunately, the OldSchoolBot does not respond to commands posted through Discord Hooks. Is there any way I could make this work?

**Comment by gc** [comment, marked as answer]
If its for controlling/interacting with your minion, definitely no - you can't, thats against the rules to automate or control your minion in any way other than you typing on a keyboard. If its not, still no, not something I support.

**Comment by gc** [comment]
Please use discord for questions like this.
